core/src/main/java/greencity/controller/UserProfileController.java

```diff
@@ -84,4 +84,21 @@ public ResponseEntity<HttpStatus> deactivateUser(
         ubsClientService.markUserAsDeactivated(id);
         return ResponseEntity.status(HttpStatus.OK).build();
     }
+
+    /**
+     * Controller return info about all locations.
+     *
+     * @return {@link ResponseEntity} list of locations.
+     * @author Max Nazaruk.
+     */
+    @ApiOperation(value = "Get info about cities in each region")
+    @ApiResponses(value = {
+        @ApiResponse(code = 200, message = HttpStatuses.OK, response = LocationSummaryDto.class),
+        @ApiResponse(code = 401, message = HttpStatuses.UNAUTHORIZED),
+        @ApiResponse(code = 403, message = HttpStatuses.FORBIDDEN)
+    })
+    @GetMapping("/getLocationSummary")
```

Comment:
please read the rest https://restfulapi.net

Reply:
Still not correct

Reply:
Could you please be more specific? What exactly is the problem of this area?